Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/ElmJxsonLibraryReader.java

```diff
@@ -1,24 +1,62 @@
 package org.cqframework.cql.cql2elm;
 
 import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.fasterxml.jackson.dataformat.xml.ser.ToXmlGenerator;
 import com.fasterxml.jackson.module.jaxb.JaxbAnnotationModule;
 import org.cqframework.cql.cql2elm.model.serialization.LibraryWrapper;
+import org.hl7.cql_annotations.r1.CqlToElmBase;
 import org.hl7.elm.r1.Library;
+import org.hl7.elm_modelinfo.r1.TypeInfo;
+import org.hl7.elm_modelinfo.r1.TypeSpecifier;
 
+import java.io.File;
 import java.io.IOException;
+import java.io.InputStream;
 import java.io.Reader;
+import java.net.URI;
+import java.net.URL;
 
 public class ElmJxsonLibraryReader {
```

Comment:
Is this class redundant now given the changes to ElmJsonLibraryReader?

Reply:
Yep, but I kept it for backward compatibility. Same for the `toJson` and `toJxson` methods in the `CQLTranslator`

Reply:
Probably fine to kill the "toJxson" methods and the redundant LibraryReader. We've had some discussions with implementors on the cql_engine repo and the consensus is that it's fine to standardize on one JSON format. JXSON was specifically added to support Android so the only user of that API is the cql-evaluator/cql-engine.

Reply:
I just removed all JXSON references in https://github.com/cqframework/clinical_quality_language/pull/752/commits/2f34b3617a5415254685759b12b88fb1a9af298c. JSON now is Jackson. Tests using JXSON only were moved to JSON. Duplicated tests were deleted.